src/sinks/sentry/metrics/sink.rs

```diff
@@ -1,17 +1,38 @@
 use async_trait::async_trait;
 use futures::{stream::BoxStream, StreamExt};
 use futures_util::future::ready;
-use sentry::metrics::Metric;
+use sentry::metrics::{Metric, MetricBuilder};
+use std::time::{Duration, UNIX_EPOCH};
 
 use crate::{
-    event::{Event, MetricValue},
+    event::{Event, Metric as SourceMetric, MetricValue},
     sinks::util::StreamSink,
 };
 
 pub struct SentryMetricsSink {
     pub dsn: String,
 }
 
+fn finish_metric(source: &SourceMetric, mut sentry_metric: MetricBuilder) {
+    if let Some(tags) = source.series().tags() {
+        for (k, v) in tags.iter_all() {
+            if let Some(v) = v {
+                sentry_metric = sentry_metric.with_tag(k.to_owned(), v.to_owned());
+            }
+        }
+    }
+
+    if let Some(time) = source.data().time.timestamp {
```

Comment:
doing this for correctness, i think it could fix some weird aggregation issues i'm seeing

Reply:
Put this in a comment?